app/views/posts/index.html.haml

```diff
@@ -0,0 +1,13 @@
+.container{:style => "font-family: Open Sans; text-align: center;"}
+  - if @posts.any?
+    - @posts.each do |post|
+      .posts
+        = link_to (image_tag post.image.url(:medium)), post_path(post), id: post.id
+        %p= post.caption
+        - if current_user
+          - if (post.user_id == current_user.id)
+            %p= link_to "Delete Post", post_path(post), method: :delete
+    %p= link_to "Add a Post", new_post_path
+  - else
+    No Pictures Yet!
+    %a(href='/posts/new') Add a Post
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))